controlpanel/frontend/jinja2/tool-list.html

```diff
@@ -9,78 +9,110 @@ <h1 class="govuk-heading-xl">Your tools</h1>
 
 <p class="govuk-body">The status of your tools will update automatically.</p>
 
-<table class="govuk-table">
-  <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th class="govuk-table__header govuk-!-width-one-half">Name</th>
-      <th class="govuk-table__header govuk-!-width-one-quarter">Status</th>
-      <th class="govuk-table__header govuk-!-width-one-quarter"><span class="govuk-visually-hidden">Actions</span></th>
-    </tr>
-  </thead>
-  <tbody class="govuk-table__body">
-  {% for tool in tools %}
-    {% set deployment = deployed_tools.get(tool) %}
-    <tr class="govuk-table__row sse-listener tool-status" data-tool-name="{{ tool.chart_name }}">
-      <td class="govuk-table__cell">
-        {{ tool.name }}
-        <br/>
-        <small class="tool-app-version">
-          {% if deployment %}
-            {{ deployment.get_installed_app_version(id_token) or "Unknown" }}
-          {% endif %}
-        </small>
-      </td>
-      <td class="govuk-table__cell">
-        <div class="tool-status-label">
-          {{ deployment and deployment.get_status(id_token) | default("") }}
+
+{# Grid Container for the list of tools (1 column, n rows) #}
+<div class="tools-list" style="display: grid; grid-template-columns: 1fr; border-top: 1px solid #b1b4b6; margin-bottom: 30px;">
```

Comment:
I think the inline CSS is something we should avoid on principle, for the sorts of reasons here https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css. I think this is implied by our frontend guidance when it mentions:

> fundamental principles like separation of style and content

https://ministryofjustice.github.io/technical-guidance/principles/frontend-development-principles/

Reply:
Fixed - that was quick!